rootfs-builder/alpine/Dockerfile.in

```diff
@@ -5,4 +5,14 @@
 
 From golang:@GO_VERSION@-alpine
 
-RUN apk update && apk add git make bash gcc musl-dev linux-headers apk-tools-static libseccomp libseccomp-dev
+RUN apk update && apk add \
```

Comment:
@marcov Will this change not be applicable to the Dockerfile for other distributions as well?

Reply:
Hi @amshinde, good point. 
Compared to the other images, alpine already have Go installed (`from golang:...`), so installing `curl` was not necessary until now. Now `curl` is needed to do the runtime check on the Go version. 

Other images already have `curl` pre-installed to download the required Go tarball. I can add curl to all of them, to be future-proof and make the dependency from it explicit. thanks!